src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -31,24 +33,30 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
         // --- end of chromosome chart ---
 
         // --- CNA bar chart ---
-        let cnaRaphaelData: any = {};
-        cnaRaphaelData[sampleId] = [];
-        _.each(this.props.cnaSegments, function(_dataObj: any) {
-            var _tmp: Array<any> = [];
-            _tmp.push(_dataObj.sample);
-            _tmp.push(_dataObj.chr);
-            _tmp.push(_dataObj.end);
-            _tmp.push(_dataObj.start);
-            _tmp.push(_dataObj.numProbes);
-            _tmp.push(_dataObj.value);
-            cnaRaphaelData[sampleId].push(_tmp);
+        _.each(uniqCnasampleIds, (_sampleId: string) => {
```

Comment:
not too important, but we might as well use the type for sampleId.  I do believe it's a type.   It may be that using array's forEach method instead of lodash will allow typescript to derive the type automatically.   I'm also meaning to see if perhaps lodash types have been improved.